pages/home.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import type { NextApplicationPage } from "./_app";
+import { AdminHomeView } from "../components/Home/AdminHomeView";
+import { NextApplicationPage } from "./_app";
 //This is the page that is rendered when the 'Home' button from the Navbar is clicked
-
 const Home: NextApplicationPage = () => {
-  return <div>Home</div>;
+  return <AdminHomeView />;
```

Comment:
To render the component based on the logged in user, something like this should work:
```typescript 
const Home: NextApplicationPage = () => {
  const { user } = useContext(AuthContext);
  if (user == null) return <Error/> // this is a component in the util directory
 const pageToRender = user.role == "Admin" ? <AdminHomeView/> : <div>Home</>;
 return ( pageToRender);
}
```